pkg/k8shandler/defaults.go

```diff
@@ -0,0 +1,34 @@
+package k8shandler
+
+import (
+	"fmt"
+)
+
+const (
+	modeUnique    = "unique"
+	modeOpsShared = "ops_shared"
+	defaultMode   = modeUnique
+)
+
+func KibanaIndexMode(mode string) (string, error) {
+	if mode == "" {
+		return defaultMode, nil
+	}
+	if mode == modeUnique || mode == modeOpsShared {
+		return mode, nil
+	}
+	return "", fmt.Errorf("Invalid kibana index mode provided [%s]", mode)
+}
+
+// 'es' or 'es-ops'
+func EsUnicastHost(opsCluster string) string {
+	return fmt.Sprintf("logging-%s-cluster", opsCluster)
+}
+
+func AllowClusterReader() bool {
```

Comment:
We actually did away with this variable when we moved to SAR based permission evals

Reply:
the outcome is that it's not present in `elasticsearch.yaml` anymore or that it's always `false`?

Reply:
its no longer read in by `openshift-elasticsearch-plugin`